src/resty/gcp/init.lua

```diff
@@ -107,8 +111,10 @@ local function build_request(accesstoken, apiDetail, baseUrl, params, requestBod
         if k == "alt" then
             query[k] = v
             goto continue
+        elseif consumed_parameters[k] then
```

Comment:
Could there be a parameter to have the location being "query" and consumed by the URL template at the same time?

Reply:
If so we need also to have a test for it.

Reply:
I've scanned the API definitions using an XSLT script and could not find any query parameters that were also listed in the path.  I think this should be good for now.